themes/bootstrap3/templates/layout/layout.phtml

```diff
@@ -119,17 +116,14 @@ JS;
     </header>
     <nav class="breadcrumbs" aria-label="<?=$this->transEscAttr('Breadcrumbs') ?>">
       <div class="container">
-        <?php if((!isset($this->layout()->showBreadcrumbs) || $this->layout()->showBreadcrumbs == true)
-          && !empty($this->layout()->breadcrumbs)
-          && $this->layout()->breadcrumbs !== false
-        ): ?>
+        <?php if (!empty($this->layout()->showBreadcrumbs) && !empty($this->layout()->breadcrumbs)): ?>
```

Comment:
I don't think this refactoring is correct, because the old code defaults showBreadcrumbs to true and the new code defaults showBreadcrumbs to false. I think you probably want `!empty($this->layout()->showBreadcrumbs ?? true)` to avoid changing behavior.

Reply:
Oh, took a while to notice but it seems like I understood that part totally wrong. Gonna tune it!